Formula/n/nwchem.rb

```diff
@@ -65,7 +72,7 @@ def install
       ENV.append "OMPI_CFLAGS", "-Wl,-ld_classic" if DevelopmentTools.clang_build_version >= 1500
       os = OS.mac? ? "MACX64" : "LINUX64"
       system "make", "nwchem_config", "NWCHEM_MODULES=all python gwmol", "USE_MPI=Y"
-      system "make", "NWCHEM_TARGET=#{os}", "USE_MPI=Y"
+      system "make", "NWCHEM_TARGET=#{os}", "USE_MPI=Y", "-j3"
```

Comment:
brew already sets this via `MAKEFLAGS`: https://github.com/Homebrew/brew/blob/e57a87cfe9c15f192cd4fc6695a2b3b7acf0ddf7/Library/Homebrew/extend/ENV/std.rb#L38

Reply:
`-j3` removed